### DISCUSSION TITLE
Save graphs

### DISCUSSION THREAD

**Discussion by ywsyws** [Kanaries/graphic-walker#47] [General]
Is there a way to save the graphs produced and no need to redrag/reproduce each time we relaunch the notebook?

**Comment by jojocys** [comment]
I am assumig you want to use save feature in pygwalker, check the pygwalker doc here: https://github.com/Kanaries/pygwalker/wiki/How-to-save-charts-in-juypter-cell-and-share-it-with-others%3F
